benchmark_data_loading/common.py

```diff
@@ -1,7 +1,7 @@
 import argparse
 import datetime
 from pathlib import Path
-from time import time
+from time import perf_counter
```

Comment:
`time()` will give you the current time stamp. While this usually works, `perf_counter` was made for benchmarking: https://docs.python.org/3/library/time.html#time.perf_counter

There is also `perf_counter_ns`, but I don't think we need that precision here.